src/app/themes/theme.js

```diff
@@ -35,24 +43,22 @@ const theme = createMuiTheme({
     },
     h3: {
       fontSize: '1.750em',
-      fontWeight: 800,
+      fontWeight: 400,
```

Comment:
Did you make these changes in the code that's going in with the Swagger config stuff? Or is it needed here in this PR? Because if it's already in the other PR it'll possibly conflict here. If it's not needed for this feature it shouldn't be here, as it makes it much harder to tell what's being used and what's not

Reply:
brought in on the Swagger Ticket

Reply:
If it's not part of this PR then it shouldn't be in this PR